gammapy/irf/background.py

```diff
@@ -411,6 +411,7 @@ def plot_spectrum(self, ax=None, **kwargs):
         energy_axis = self.axes["energy"]
 
         bkg = self.integral(offset=offset_axis.bounds[1], axis_name="offset")
+        bkg = bkg.to(u.Unit("s-1 MeV-1"))
```

Comment:
@registerrier , should we hardcoded MeV? Why not TeV?

Reply:
As in the future we have to support more formats than gadf, one could consider that the energy axis unit of the bkg IRF is a good guess.
So `bkg.to(Unit("s-1")/energy_axis.unit)`  could work.